src/components/Box/Box.module.scss

```diff
@@ -0,0 +1,16 @@
+.box {
```

Comment:
instead of making this a box css module, should it be a transition utility?

Reply:
yes, I think that would be ideal, I just felt that the scope of the pr was getting a bit away from me with adding focus and cursor utilities on top of hover.

I wasn't 100% sure how to best move forward with transition utilities given that they should probably allow transition of specific properties and not necessarily across the board when applied.